src/WindowsAzureMessaging/WindowsAzureMessaging/include/WindowsAzureMessaging/MSNotificationHubDelegate.h

```diff
@@ -18,9 +18,29 @@
  *
  * @param notificationHub The instance of MSNotificationHub
  * @param message The push notification details.
+ * @param completionHandler The completion handler which should be called with NoData upon completion.
  */
 - (void)notificationHub:(MSNotificationHub *)notificationHub
     didReceivePushNotification:(MSNotificationHubMessage *)message
         fetchCompletionHandler:(void (^)(UIBackgroundFetchResult))completionHandler;
 
+/**
+ * Callback method that will be called whenever the push token is retrieved.
+ *
+ * @param notificationHub The instance of MSNotificationHub
+ * @param pushToken The push token
+ */
+- (void)notificationHub:(MSNotificationHub *)notificationHub 
```

Comment:
If we're going to do a rename, we should have maybe a sample linked in our Swift sample app?